cypress/utils/index.ts

```diff
@@ -1,55 +1,84 @@
 import Web3 from "web3";
 import PrivateKeyProvider from "truffle-privatekey-provider";
+import * as BigNumber from 'bignumber.js';
 
 // this is exactly the same as ganache.sh file in localnode
 const ACCOUNT_1_PRIV = "0x47be2b1589bb515b76b47c514be96b23cd60ee37e81d63c2ae9c92f7d7667e1a";
 export const ACCOUNT_3_PRIV = "0x1ff8271bf14ac9bef0b641cced40dc2a7ebd2e37d8e16d25b4aa1911364219af";
+export const ACCOUNT_3_PUBLIC = "0x79d7176ae8f93a04bc73b9bc710d4b44f9e362ce";
 
-export let web3;
+export let web3: any;
```

Comment:
Does it add any value?

Reply:
it is required by the compiler (explicit anys) i migrate whole file to TS